packages/cache/src/cache.ts

```diff
@@ -119,9 +119,16 @@ function getExpirationTtl(
   const originalNow = CachePolicy.prototype.now;
   // @ts-expect-error `now` isn't included in CachePolicy's type definitions
   CachePolicy.prototype.now = clock;
+
+  const originalStorable = CachePolicy.prototype.storable;
   try {
     const policy = new CachePolicy(cacheReq, cacheRes, { shared: true });
 
+    if (cf?.cacheEverything) {
+      delete resHeaders["set-cookie"];
+      CachePolicy.prototype.storable = () => true;
+    }
```

Comment:
@mrbbot I decided to make it this way after reading this part:

https://github.com/kornelski/http-cache-semantics/blob/c20dc7eeca608339143857953ee7ed9343fe6d3d/index.js#L389-L454

and public documentation of Cloudflare (which lacks details, or I haven't found them)

It means it may not replicate actual behavior of Cloudflare and it won't even replicate the actual behavior of `CachePolicy` in the following aspects:
- `vary: *` will render TTL: 0
- `cache-control: no-cache` will render TTL: 0
- `cache-control: proxy-revalidate` will render TTL: 0 (this one is an actual "wait, it should be discussed" )

I don't know if it's good enough, please advise